config/jobs/kubernetes-sigs/cluster-api/cluster-api-upgrades-main.yaml

```diff
@@ -0,0 +1,174 @@
+periodics:
```

Comment:
Lets call this file cluster-api-periodics-main-upgrades.yaml so it will be presented near cluster-api-upgrades-main.yaml